civcraftConfigs/gimmelconfig.yml

```diff
@@ -222,6 +222,7 @@ monster:
       pistonwithstickyblood:
        material: PISTON_STICKY_BASE
        amount: 1
+       chance: 1
```

Comment:
for some reason the sticky pistons were not dropping so were tryin to figure out why, so adding a chance as 100% we thought could force it to drop the pistons. if not ill change it out but we still need to find why its not dropping them